src/core/help/dialog/HelpDialog.tsx

```diff
@@ -53,41 +46,77 @@ const CustomNewIcon = styled(FiberNewTwoToneIcon)(() => ({
 
 const HelpDialog = ({ open, onClose }: HelpDialogProps) => {
   const { t } = useTranslation();
+
   const { locations } = useConfig();
+
+  const { services } = useServerMetadata();
+
   const handleClose = () => (onClose ? onClose() : undefined);
-  const faq = `${locations?.help}`;
-  const contactUs = `${locations?.support}`;
-  const gettingStarted = `${locations?.help}/getting-started`;
+
+  const supportHref = locations?.support;
+  const docsHref = `/${TopLevelRoutePath.Docs}`;
+  const welcomeSpaceHref = `/${TopLevelRoutePath.WelcomeSpace}`;
 
   return (
-    <Dialog open={open} onClose={handleClose} maxWidth={'xl'}>
+    <Dialog open={open} onClose={handleClose} sx={{ '& .MuiDialog-paper': { maxWidth: 700 } }}>
       <DialogHeader title={t('pages.help-dialog.title')} onClose={handleClose} />
       <HelpDialogContent>
         <WrapperMarkdown>{t('pages.help-dialog.text')}</WrapperMarkdown>
         <Grid container columns={{ xs: 4, sm: 6 }}>
           <Grid item xs={2}>
-            <IconWrapper href={faq} target="_blank">
-              <Icon component={QuizOutlinedIzon} />
-              {t('pages.help-dialog.icons.help-center-faq')}
+            <IconWrapper
+              href={docsHref}
+              target="_blank"
+              rel="noopener"
+              aria-label={t('pages.help-dialog.icons.exploreDocumentation')}
+            >
+              <Icon component={QuizOutlinedIcon} />
+              {t('pages.help-dialog.icons.exploreDocumentation')}
             </IconWrapper>
           </Grid>
+
           <Grid item xs={2}>
-            <IconWrapper href={contactUs} target="_blank">
+            <IconWrapper
+              href={supportHref}
+              target="_blank"
+              rel="noopener"
+              aria-label={t('pages.help-dialog.icons.contactTheTeam')}
+            >
               <Icon component={ForumOutlinedIcon} />
-              {t('pages.help-dialog.icons.community-support-forum')}
+              {t('pages.help-dialog.icons.contactTheTeam')}
             </IconWrapper>
           </Grid>
+
           <Grid item xs={2}>
-            <IconWrapper href={gettingStarted} target="_blank">
+            <IconWrapper
+              href={welcomeSpaceHref}
+              target="_blank"
+              rel="noopener"
+              aria-label={t('pages.help-dialog.icons.joinTheWelcomeSpace')}
+            >
               <Icon component={CustomNewIcon} />
-              {t('pages.help-dialog.icons.new-user')}
+              {t('pages.help-dialog.icons.joinTheWelcomeSpace')}
             </IconWrapper>
           </Grid>
         </Grid>
-        <FooterLink href={locations?.tips} target="_blank">
-          <ArrowForwardIcon />
-          {t('pages.help-dialog.tips-and-tricks')}
-        </FooterLink>
+
+        <Box sx={{ display: 'flex', justifyContent: 'center', marginTop: gutters(2) }}>
```

Comment:
Information: We tend to use our wrappers over the default MUI components. So we can either use Grid, Gutters, Actions, etc. instead of Box.

In this case you can use <Gutters row> wich automatically will give gap to the items below, no need to define margins.